_pages/getting-started/tts-classes.md

```diff
@@ -0,0 +1,105 @@
+---
+title: 'TTS Classes for TTS Onboarding'
+tags:
+- onboarding
+- getting started
+- tts classes
+---
+
+We’ve put together a curriculum of classes we encourage you to take during your [first 90 days](https://handbook.tts.gsa.gov/getting-started/).
+
+These classes will expose you to a variety of aspects of our organization, introduce you to the people involved, and create opportunities to ask questions.
+
+**Most of these classes are recommended, but optional** (with a few exceptions). If you have a conflicting team meeting or another commitment, prioritize that first! You can read the materials at your convenience or join the classes 2 weeks later.
+
+**There is A LOT to take in over your first 1-3 months.** So take it all at your own pace, ask questions, and feel free to follow up with your class leads at any point — they’re happy to help.
+
+## Curriculum
+
+This section provides an overview of each of the classes, broken into topics, followed by a [a calendar of the first two weeks of classes]({{site.baseurl}}/tts-classes/#calendar).
+
+### Topics
+
+* [Working in the Government]({{site.baseurl}}/tts-classes/#working-in-the-government)
+* [Background and Operations at TTS]({{site.baseurl}}/tts-classes/#background-and-operations-at-tts)
+* [Our Values and Principles at TTS]({{site.baseurl}}/tts-classes/#our-values-and-principles-at-tts)
```

Comment:
Minor thing, but I think it would be a nice touch to lead with values and principles.